base/math.jl

```diff
@@ -263,14 +263,29 @@ cbrt(x::AbstractFloat) = x < 0 ? -(-x)^(1//3) : x^(1//3)
 """
     exp2(x)
 
-Compute ``2^x``.
+Compute the base `2` exponential of `x`, in other words ``2^x``.
```

Comment:
I don't see a harm in highlighting the number, maybe others have  a different thought on this? Will update accordingly.